src/services/users/signin-user-by-email.test.js

```diff
@@ -0,0 +1,51 @@
+const { expect } = require('chai')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const signInUserByEmail = require('./signin-user-by-email')
+
+describe('User signin', () => {
+	let user
+	let validPassword
+	beforeEach(async () => {
+		user = await fixtures.insertUser({
+			password : 'validPaswword123$'
+		})
+		validPassword = 'validPaswword123$'
+	})
+
+	it('should throw an error if user does not exist', async () => {
+		const { id,email } = user
+		await db.query(
+			`DELETE FROM users
+             WHERE id = $1`, [id])
```

Comment:
forgot to flag this, let's use a made-up email instead of deleting the user. 

we might probably just need `before` instead of a `beforeEach` if we do that